packages/plugin-doc-coverage/src/lib/doc-coverage-plugin.ts

```diff
@@ -0,0 +1,54 @@
+import type { PluginConfig } from '@code-pushup/models';
+import {
+  type DocCoveragePluginConfig,
+  docCoveragePluginConfigSchema,
+} from './config.js';
+import { PLUGIN_SLUG, groups } from './constants.js';
+import { createRunnerFunction } from './runner/runner.js';
+import {
+  filterAuditsByPluginConfig,
+  filterGroupsByOnlyAudits,
+} from './utils.js';
+
+export const PLUGIN_TITLE = 'Documentation coverage';
+
+export const PLUGIN_DESCRIPTION =
+  'Official Code PushUp documentation coverage plugin.';
+
+export const PLUGIN_DOCS_URL =
+  'https://www.npmjs.com/package/@code-pushup/doc-coverage-plugin/';
```

Comment:
```suggestion
const PLUGIN_TITLE = 'Documentation coverage';

const PLUGIN_DESCRIPTION =
  'Official Code PushUp documentation coverage plugin.';

const PLUGIN_DOCS_URL =
  'https://www.npmjs.com/package/@code-pushup/doc-coverage-plugin/';
```

If we dont use them somewhere else we could remove the export

Reply:
I used it in the unit.test, but I guess we should not modify logic files for testing sake besides exporting functions... Mmm, what do you think?

Reply:
Also I just tried for science's sake what happens if you provide a wrong one, and being everything based on includes and filter, there is no issue, it just ignores it, maybe it would be interesting to throw a warning like that audit does not exist? what do you think?